app/assets/javascripts/groups_index.js

```diff
@@ -37,6 +40,26 @@ if (!top.location.pathname.includes("users")) {
     }
   }
 
+  function leave_group(group_id) {
+    $button.removeClass("is-member");
+    $button.removeClass("remove-member");
+    $button.text("Join Group");
+    $.ajax({
+      type: "PUT",
+      url: "/groups/" + $button.attr("value") + "/leave",
+      success: function(data) {
+        $(".group_id" + group_id).removeClass("active");
```

Comment:
don't repeat yourself in your code --   `$(".group_id" + group_id)` can be set to a variable. `$group_div = $(".group_id" + group_id)`